CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+
+## Propose a Feature
+
+Open a issue, mark as other, and describe the current scenario, it's problems, and how your resource solves a problem or introduce new functionalities.
+
+## Choose a Task on the Project
+
+Choose a task in the [project](https://github.com/agrotis-io/vraw/projects/1), read the description of the issue and if you have any questions, leave a comment.
+
+## Issue Labeling
+
+To identify what kind of issue you are planning to work, or to check our list of tasks and issues in more detail take a look at this table:
+
+| Label            | Description                                                        |
+| ---------------- | ------------------------------------------------------------------ |
+| docs             | Indicate documentation development necessary                       |
+| feat             | Indicate feature development                                       |
+| CI               | Indicate Continuous integration development                        |
+| refactor         | Indicate code refactor                                             |
+| remove           | Indicate feature removal                                           |
+| fix              | Indicates an unexpected problem or unintended behavior             |
+| duplicate        | Indicates similar issues or pull requests                          |
+| enhancement      | Indicates new feature requests                                     |
+| good first issue | Indicates a good issue for first-time contributors                 |
+| help wanted      | Indicates that a maintainer wants help on an issue or pull request |
+| invalid          | Indicates that an issue or pull request is no longer relevant      |
+| question         | Indicates that an issue or pull request needs more information     |
+| wontfix          | Indicates that work won't continue on an issue or pull request     |
+
+## Fork or Clone It
+
+In order to start colaborating to our project, you have to **FORK** this repository if you are a contributor, or **CLONE** it if you are a not a contributor.
+
+## Development Workflow
+
+This will help you understand how you should develop your contribution to the repository:
+
+![Repository Workflow](images/workflow.png)
+
+## Install Dependencies
+
+After getting the repository locally in your machine, you have to install all the dependencies in order to run and start contributing.
+
+To install everything you need, go to the folder you cloned the repository and run: _yarn install_
+
+## Testing your Component
+
+If your contribution is a new component, or a modification in an existing one, you might test it's final UI version with Storybook.
+
+To test the component, type this command in the project's folder location: _yarn run storybook_
```

Comment:
show the command like this

```bash
yarn run storybook
```

its better then "_yarn run storybook_" in a big doc